src/main/scala/is/hail/variant/Locus.scala

```diff
@@ -53,26 +50,26 @@ object Locus {
   }
 
   object LocusIntervalParser extends JavaTokenParsers {
-    def parseInterval(input: String, gr: GRBase): Interval[Locus] = {
-      parseAll[Interval[Locus]](interval(gr), input) match {
+    // FIXME test against gr
+    def parseInterval(input: String, gr: GRBase): Interval = {
+      parseAll[Interval](locus_interval, input) match {
         case Success(r, _) => r
         case NoSuccess(msg, next) => fatal(s"invalid interval expression: `$input': $msg")
       }
     }
 
-    def interval(gr: GRBase): Parser[Interval[Locus]] = {
-      implicit val locusOrd = gr.locusOrdering
+    // FIXME make end correct
+    def locus_interval: Parser[Interval] = {
```

Comment:
This is fine for now, but my PR will need the GRBase. I check the Locus has valid contig and pos. I also fixed the end in my branch.

Reply:
OK, great.  Or if your PR is ready to go in, I'm happy to let it go in first and rebase.

Reply:
Your call. @danking needs to review it again #2558.